internal/command/run.go

```diff
@@ -50,7 +50,7 @@ func Run(ctx context.Context, cmd *cli.Command) error {
 	printStartUpMessage()
 
 	// get config
-	cfg, err := config.New(env)
+	cfg, err := config.New(env, cmd)
```

Comment:
Changing the method signature of `config.New` is going to break a log of things.

This is likely to get reworked very soon, see #379 , so I'd recommend something like this which will keep the change minimal:

```go
	cfg, err := config.New(env)

    // check err ...

    if cmd.String("plan-identifier") != "" {
      cfg.Identifier = cmd.String("plan-identifier")
    }
```

basically set this directly here rather than in `Config.New`.

Reply:
I originally had code you suggested but thought it would be better to consolidate the creation of the config by passing through the CLI flags. But with the extra context that's all going to change, I'll revert back to the minimal change.